docs/howto/create-a-release.rst

```diff
@@ -102,6 +102,115 @@ Upload build to PyPI
+As there are inter-dependencies amongst the Cylc 8 parts, you should
+know the dependency tree, and start by the leaf nodes (i.e. a module
+A may have multiple dependants, but no dependency to other modules).
+
+For example, Cylc UI Server depends on Cylc Flow. So unless you are
+releasing only Cylc UI Server, you should release Cylc Flow first.
+
+* On GitHub, navigate to the project repository on GitHub, e.g.
+
+- https://github.com/conda-forge/metomi-isodatetime-feedstock
+- https://github.com/conda-forge/cylc-uiserver-feedstock
+- https://github.com/conda-forge/cylc-ui-feedstock
+- https://github.com/conda-forge/cylc-flow-feedstock
+- https://github.com/conda-forge/cylc-feedstock
+
+On each of these repositories, the release process should be pretty
+much the same. Your work will be mainly (if not exclusive) on
+the ``recipe/meta.yaml`` file.
+
+This is a possible order of action:
+
+- Check open pull requests—There are bots that alert about dependencies
+(N.B. conda packages do not use PYPI for dependencies, they all use
+other conda packages as dependencies!.
+- Check if you need to update the version in the Jinja2 variable right
+at the top of the file.
+- If you updated the version, set the build number back to 0.
+- If you did not update the version, increase the build number.
+- If the conda package contains a source URL, check that it is evaluated
+correctly after the version is replaced, and update the ``sha256`` value.
+- Check requirements (we may have changed setup.py, or package.json, etc.
+ensure we are up to date).
+- Ensure the test commands are still valid
+- Check if you need to change summary, description, or any other field.
+- Create a branch in your clone, commit, and push to your fork. Wait
+for the feedback of the Conda Forge CI bots.
+- In the meantime, test the package locally (see below).
+- If the bots gave you the OK, and testing locally worked fine too,
+your pull request should be ready for review.
+- If you changed something in the recipe, and you are not sure if
+that will affect the Conda Forge package, you can ask Conda Forge
+maintainers help reviewing the pull request (via Gitter).
+- Otherwise, get others from Cylc Core team to review & approve. Once
+your pull request has been approved and merged, the bots will start the
+release process.
+
+From past experience, I've seen Conda packages available in Conda Forge
+in under 30 minutes, but have also had to wait hours (<8 hours) for it
+to show in Conda Forge. For example, if Azure or GitHub infrastructure
+have issues (as I experienced during the first releases) it may take days.
+So keep that in mind before announcing the Conda packages.
+
+To test packages locally, first you should make sure that Conda is
+configured to avoid automatically upload the package. Open you ``~/.condarc``
+and check that you have something similar to:
+
+.. code-block:: yaml
+    channels:
+      - defaults
+      - conda-forge
+    ssl_verify: true
+    anaconda_upload: false
```

Comment:
```suggestion
.. code-block:: yaml

   channels:
     - defaults
     - conda-forge
   ssl_verify: true
   anaconda_upload: false
```
Sorry - rst is terribly picky about whitespace but I think that this should fix it - check at
https://github.com/kinow/cylc-admin/blob/conda-forge-instructions/docs/howto/create-a-release.rst